app/_hub/kong-inc/rate-limiting/2.1-x.md

```diff
@@ -0,0 +1,233 @@
+---
+name: Rate Limiting
+publisher: Kong Inc.
+redirect_from:
+  - /enterprise/0.35-x/rate-limiting/
```

Comment:
is this redirect from correct?

Reply:
Yes, that was when the enterprise version of the plugin was renamed to `rate-limiting-advanced`